tests/utils/__snapshots__/getAdminSub.test.js.snap

```diff
@@ -7,7 +7,7 @@ Array [
     "UserAttributes": Array [
       Object {
         "Name": "email",
-        "Value": "cellenics_admin@listserv.med.harvard.edu",
+        "Value": "admin@biomage.net",
```

Comment:
similar question as above - is it reasonable to eliminate the need for organisation name within these snaps